components/sso-hostobject/org.wso2.carbon.hostobjects.sso/src/main/java/org/wso2/carbon/hostobjects/sso/internal/builder/LogoutRequestBuilder.java

```diff
@@ -100,4 +153,113 @@ public LogoutRequest buildLogoutRequest(String subject, String reason,
 
         return logoutReq;
     }
+
+    /**
+     * Overload Logout request for sessionIndexId is not exist case
+     * @param subject
+     * @param reason
+     * @param issuerId
+     * @return
+     */
+    public LogoutRequest buildSignedLogoutRequest(String subject, String reason,
+            String issuerId, int tenantId, String tenantDomain, String destination) throws Exception {
```

Comment:
Fixed generic exception throwing in #1528 (https://github.com/wso2/carbon-apimgt/commit/c6d6cb237bd7654eda952fb2fbd74863e9899cfc)